aws-ecsfargate-terraform/main.tf

```diff
@@ -80,7 +115,7 @@ resource "aws_secretsmanager_secret" "scimsession" {
 
 resource "aws_secretsmanager_secret_version" "scimsession" {
   secret_id     = aws_secretsmanager_secret.scimsession.id
-  secret_string = base64encode(file("${path.module}/scimsession"))
+  secret_string = filebase64("${path.module}/scimsession")
```

Comment:
Nice, is `filebase64` a terraform specific function?

Reply:
Yep! (See [`filebase64`](https://www.terraform.io/language/functions/filebase64).)